applications/wg-easy/.env

```diff
@@ -0,0 +1,4 @@
+VERSION=0.0.4
+CHANNEL=wg-easy
+APP_NAME=wg-easy
+RELEASE_NOTES="Release created via task release-create"
```

Comment:
This also feels like a file that you shouldn't have checked in. Aren't all of these values set by defaults already?

Reply:
Those values has been set as default as var
```
EMBEDDED: '{{.EMBEDDED | default "false"}}'
LICENSE_ID: '{{if eq .EMBEDDED "true"}}{{.LICENSE_ID | default "2cmqT1dBVHZ3aSH21kPxWtgoYGr"}}{{end}}'
TIMEOUT: '{{if eq .EMBEDDED "true"}}420{{else}}300{{end}}'
```
It is not flexible when I copy this taskfile and apply to my own app. I am trying to migrate those values back to .env.

I will remove .env and put it into some session of README.md